scripts/actions/mobskills/rotten_stench.lua

```diff
@@ -0,0 +1,31 @@
+-----------------------------------
+-- Rotten Stench
+-- Lowers physical and magic accuracy of enemies in an area of effect.
+-----------------------------------
+local mobskillObject = {}
+
+mobskillObject.onMobSkillCheck = function(target, mob, skill)
+    return 0
+end
+
+mobskillObject.onMobWeaponSkill = function(target, mob, skill)
+    local power        = 50
+    local duration     = math.random(90, 120)
```

Comment:
I recently added a new function for this. To be honest I'm not 100% positive this is correct for all mobskills but it seems more likely than a random for duration.
```lua
local duration = xi.mobskills.calculateDuration(skill:getTP(), 90, 120)
```